Internal reverts with reason in debug mode (#8178)
ethereum/solidity: Changelog.md

Compiler Features:
 * Code Generator: Use ``calldatacopy`` instead of ``codecopy`` to zero out memory past input.
 * AST: Add a new node for doxygen-style, structured documentation that can be received by contract, function, event and modifier definitions.

>  * Debug: Inject strings for compiler-generated internal reverts when using the ``--revert-strings`` option or the ``settings.debug.revertStrings`` setting on ``debug`` mode.

Reviewer: ```suggestion
 * Debug: Provide reason strings for compiler-generated internal reverts when using the ``--revert-strings debug`` / ``settings.debug.revertStrings = "debug"`` option.
```
